### PR TITLE
Add __repr__ for objects

### PR DESCRIPTION
**Pull request by dgusakov**

### CODE REVIEW

**Comment by dgusakov** [comment]
@a-alferov @skhomuti  I'm not sure about this reprs. We definitely need them, but maybe in some other way.

File: adcm_client/objects.py [hunk 1]
```diff
@@ -650,6 +668,9 @@ class Action(BaseAPIObject):
     subs = None
     config = None
 
+    def __repr__(self):
+        return f"<Action {self.name} at {id(self)}>"
```
Comment (skhomuti) [inline adcm_client/objects.py:672]: There is no info about parent object? Cluster, service, or host

Comment (dgusakov) [reply adcm_client/objects.py:672]: There is pretty complicated object hierarchy for Action, Task and Job objects. According to existing class code there is no simple way to get parent object.

**Comment by skhomuti** [comment]
https://github.com/allure-framework/allure-python/blob/master/allure-python-commons/src/utils.py#L115 - how Allure shows parameters in the log

**Review by a-alferov — APPROVED** [review]

**Review by skhomuti — APPROVED** [review]

**Pull request merged**